p2p/src/net/mock/types.rs

```diff
@@ -193,6 +193,8 @@ pub enum MockEvent {
     SendMessage(Box<Message>),
 }
 
+pub type HandshakeNonce = u128;
```

Comment:
I think a u64 is more than enough for this purpose. This doesn't have to be cryptographically secure.

Reply:
Sure, updated.